### PR TITLE
Added 'start over' button, issue # 100

### PR DESCRIPTION
**Pull request by lucasoooooo**

### CODE REVIEW

**Comment by lucasoooooo** [comment]
Hi @doubleiis02 , made a quick edit to Pathway.js to add a "start over" button

File: src/pages/Pathway/Pathway.js [hunk 1]
```diff
@@ -23,7 +23,10 @@ const Pathway = () => {
             {example_resources.map((resource, index) => 
                 <PathDay index={index + 1} resource={resource} />
             )}
-            
+            <h2>Come back later?</h2>
+            <p3>You can come back later and your pathway will be saved! Alternatively,
+                 you can <a href="/quiz" target="_blank" rel="noreferrer noopener">start over</a>.</p3>
```
Comment (doubleiis02) [inline src/pages/Pathway/Pathway.js:28]: could we remove the `target="_blank"`? we don't actually want to open a new tab when we start over D:

**Review by doubleiis02 — CHANGES_REQUESTED** [review]
hey thanks for the short fix! i requested one super single change but other than that looks good :)

and sorry i should have made this clear before leaving these issues out in the open, but the dev pathways issues are being reserved for ACM Teach LA's internal use (for teaching purposes), so it would be great if any tickets with the `reserved` labels are not worked on by outside contributors. feel free to check out any of ACM's other open-source projects though!! thanks!!

File: src/pages/Pathway/Pathway.js (hunk 1 above)
Comment (lucasoooooo) [reply src/pages/Pathway/Pathway.js]: Hello, sorry about that, I must have been the reserved label. I was trying to contribute to an open source project for my class, but ill make sure to check out other ACM's open-source projects!
Anyways made the fix and deleted target="_blank".
Thanks!

**Review by doubleiis02 — APPROVED** [review]
LGTM, feel free to squash and merge!

**Comment by lucasoooooo** [comment]
> LGTM, feel free to squash and merge!

I do think I have write access to the repo

**Comment by doubleiis02** [comment]
ah right, i'll just do it then

**Pull request merged**